platform/lib/platform_web/components.ex

```diff
@@ -1433,11 +1426,7 @@ defmodule PlatformWeb.Components do
         </span>
         <span class="ml-4 flex-shrink-0">
           <%= if Permissions.can_edit_media?(@current_user, @media, @attr) and not @immutable do %>
-            <%= live_patch("Update",
-              class: "text-button mt-1 inline-block",
-              to: Routes.media_show_path(@socket, :edit, @media.slug, @attr.name),
-              replace: true
-            ) %>
+            <.link method = "Update" class = "text-button mt-1 inline-block" patch = {Routes.media_show_path(@socket, :edit, @media.slug, @attr.name)} replace = {true} ></.link>
```

Comment:
This is incorrect. That first argument in `live_patch` is to set the label; `method = "Update"` doesn't do this, and now the button has no label. See https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#link/1.

<img width="652" alt="image" src="https://github.com/atlosdotorg/atlos/assets/3037552/88f374a8-94bf-4e6b-93a7-f6851bba4ecb">